RuddockWebsite/modules/account/templates/create_account.html

```diff
@@ -43,8 +43,8 @@
       <td>{{ user_data['email'] }}</td>
     </tr>
     <tr>
-      <td>Birthday (YYYY-MM-DD)</td>
-      <td><input type="text" name="birthday" /></td>
+      <td>Birthday</td>
+      <td><input type="text" name="birthday" placeholder="YYYY-MM-DD" /></td>
```

Comment:
This might be useful to force client-side validation:

<input type="text" pattern="\d{2}/\d{2}/\d{4}" placeholder="YYYY-MM-DD" />